Money_transfer_exercise/Account.java

```diff
@@ -0,0 +1,73 @@
+package Money_transfer_exercise;
+
+public class Account {
+	private String id;
+	private String name;
+	private int balance;
+
+	public Account(String id, String name, int balance) {
+		this.id = id;
+		this.name = name;
+		this.balance = balance;
+	}
+
+	public String getId() {
+		return id;
+	}
+
+	public String getName() {
+		return name;
+	}
+
+	public int getBalance() {
+		return balance;
+	}
+
+	public void credit(int amount) {
+		if (checkParam(amount)) {
+			this.balance += amount;
+			System.out.println("Credit successfully");
+		} else {
+			System.out.println("Credit unsuccessfully");
+		}
```

Comment:
Mạnh dạn thêm return nhé. để mục tiêu là trả về kết quả sớm nhất nếu có thể đồng thời bớt thèn `else` 
```suggestion
if (checkParam(amount)) {
			this.balance += amount;
			System.out.println("Credit successfully");
			return;
		}
		System.out.println("Credit unsuccessfully");
```

Reply:
tương tự cho transfer và debit em cũng thử thêm return vào nhé 
ko dc thì mạnh dạn  chị hì